floss/main.py

```diff
@@ -200,9 +205,11 @@ def make_parser(argv):
         type=str,
         choices=[l.value for l in Language if l != Language.UNKNOWN],
         default=Language.UNKNOWN.value,
-        help="use language-specific string extraction, auto-detect language by default, disable using 'none'"
-        if show_all_options
-        else argparse.SUPPRESS,
+        help=(
```

Comment:
are these due to missing updates in the elf-support branch that we can resolve by rebasing/updating to master?

Reply:
Sure let me check that

Reply:
The formatting were changed after I have applied the precommit linters on the file. This is matching the master so should i just keep this chnages?

Reply:
can you rebase then please?

Reply:
Sure will do that at after you are fine with these changes.

Reply:
I've updated the elf-support branch with recent master updates.

Reply:
can you rebase this branch to `elf-support`?

Reply:
rebasing/cleaning up the history TODO then this should be good to go